src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -26,6 +26,12 @@ public class StrykerOptions
         /// </summary>
         public string ProjectPath { get; init; }
 
+        /// <summary>
+        /// When true, stryker is mutating all projects in a solution
+        /// </summary>
+        /// <returns></returns>
+        public bool IsSolutionContext => SolutionPath != null && FilePathUtils.NormalizePathSeparators(ProjectPath) == FilePathUtils.NormalizePathSeparators(Path.GetDirectoryName(SolutionPath));
```

Comment:
This is not an option, it does not belong here

Reply:
It kinda is an option. The user decides if the run is a solution run or not. They don't specify it by passing a value but by the working directory they run from. I don't know a better place to put this

Reply:
I know you don't agree with this solution. But can we create a new issue to fix the size of the options class? I feel this PR is big enough as it is